src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/Summary.tsx

```diff
@@ -49,27 +49,42 @@ export function Summary({
     .join('/');
 
   const SummaryText = () => {
+    const summary = (text: string) => (
+      <ThemeText type="body__secondary" style={styles.message}>
+        {text}
+      </ThemeText>
+    );
     switch (fareProductTypeConfig.configuration.zoneSelectionMode) {
       case 'multiple':
       case 'multiple-stop':
       case 'multiple-zone':
       case 'single':
       case 'single-stop':
       case 'single-zone':
+        return summary(t(PurchaseOverviewTexts.summary.messageInZone));
+      case 'multiple-stop-harbor':
+        const harborText = summary(
+          t(PurchaseOverviewTexts.summary.messageInHarborZones),
+        );
+        if (fareProductTypeConfig.type === 'boat-period')
+          return (
+            <>
+              {summary(t(PurchaseOverviewTexts.summary.messageInHarborPeriod))}
+              {harborText}
+            </>
+          );
         return (
-          <ThemeText type="body__secondary" style={styles.message}>
-            {t(PurchaseOverviewTexts.summary.messageInZone)}
-          </ThemeText>
+          <>
+            {summary(t(PurchaseOverviewTexts.summary.messageInHarborSingle))}
+            {harborText}
+          </>
```

Comment:
Could be simplified
````
 return (
          <>
            {fareProductTypeConfig.type === 'boat-period' ? summary(t(PurchaseOverviewTexts.summary.messageInHarborPeriod)) : summary(t(PurchaseOverviewTexts.summary.messageInHarborSingle))}
            {harborText}
          </>
        );

Reply:
Maybe even `{summary(t(PurchaseOverviewTexts.summary[fareProductTypeConfig.type === 'boat-period' ? "messageInHarborPeriod" : "messageInHarborSingle"]))}`?